lib/media/media_source_capabilities.js

```diff
@@ -27,6 +27,17 @@ shaka.media.Capabilities = class {
     supportMap.set(type, currentSupport);
     return currentSupport;
   }
+
+  /**
+   * Determine support for SourceBuffer.changeType
+   * @return {boolean}
+   */
+  static isChangeTypeSupported() {
+    // eslint-disable-next-line no-restricted-syntax
+    return SourceBuffer !== undefined && SourceBuffer.prototype !== undefined &&
```

Comment:
If SourceBuffer is not globally defined, this will throw.  Please change to something like:

```js
!!window.SourceBuffer && !!SourceBuffer.prototype && !!SourceBuffer.prototype.changeType
```